pkg/get/get_test.go

```diff
@@ -1185,6 +1185,36 @@ func Test_DownloadEKSCTL(t *testing.T) {
 	}
 }
 
+func Test_DownloadEKSCTLANYWHERE(t *testing.T) {
+	tools := MakeTools()
+	name := "eksctl-anywhere"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v0.12.0"
+
+	tests := []test{
```

Comment:
There are a bunch of missing unit tests here given how many downloads you have in the template?

Reply:
The template that you've added seems to have 3 different variations, so you may need to remove one,?